pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -17,7 +17,7 @@ let
     , wrapperArgs ? []
     , manifestRc ? null
     , withPython2 ? false
-    , withPython3 ? true,  python3Env ? null
```

Comment:
I think the right fix here would be to error if python3Env is null. Overriding inputs to null is an anti pattern.

Reply:
But it's an optional feature, isn't it? My understanding is that you can provide `python3Env` to have Python plus some extra modules available. If I don't provide one, it uses the default one? Otherwise I guess the default value of python3Env should be `python3` instead of `null`?

Reply:
@SuperSandro2000  I have kept the boolean but used a sane default value for python3Env as @teto has suggested

Reply:
> But it's an optional feature, isn't it?

Yes but since nix is lazily evaluted it is never actually build when behind lib.optional(s)/ if then else.